cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -82,8 +82,8 @@ func (dc *deleteClusterOptions) validate(ctx context.Context, args []string) err
 	}
 
 	kubeconfigPath := getKubeconfigPath(clusterConfig.Name, dc.wConfig)
-	if !validations.FileExistsAndIsNotEmpty(kubeconfigPath) {
-		return kubeconfig.NewMissingFileError(kubeconfigPath)
+	if err := validations.KubeConfigFile(kubeconfigPath); err != nil {
```

Comment:
This still validates whether the data contained in the file matches a kubeconfig structure. We should probably name the method as such? validations.KubeConfigFile is confusing as to what this is supposed to perform.

Reply:
I'm confused as to what you're confused about? The function clearly states that it's validating the kubeconfig file. If you want to know exactly what that entails, it's very easy to look at the godoc for the function (your editor probably displays is somewhere either on mouseover, or via a handy keybinding) and if that fails, you have the source.

Putting the specific validations in the name gets unwieldy very quickly. For example, this function would be named: ValidateFileExistsAndIsNotEmptyAndHasValidYAMLKubeconfigStructure or something like that.